etna/core/mixins.py

```diff
@@ -13,6 +12,7 @@
 from typing import Tuple
 from typing import cast
 
+import dill
```

Comment:
Hm, this is a breaking change potentially

Reply:
Potentially, yes, but current tests are passing, so I hope it will be fine.

I added this only because with new version of `holidays` library saving/loading doesn't work for `HolidaysTransform`.

Reply:
Interestingly, if I run tests locally, tests for `HolidayTransform` are working fine. They fail only on github.

Reply:
May be you have some ideas about the error: [ci](https://github.com/tinkoff-ai/etna/actions/runs/5059352567/jobs/9080756659).

Reply:
I created the [issue](https://github.com/dr-prodigy/python-holidays/issues/1236).